modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web/src/main/java/com/liferay/portal/search/tuning/synonyms/web/internal/index/SynonymSetIndexReaderImpl.java

```diff
@@ -99,10 +100,10 @@ private Optional<Document> _getDocumentOptional(
 			getDocumentRequest);
 
 		if (getDocumentResponse.isExists()) {
-			return Optional.of(getDocumentResponse.getDocument());
+			return getDocumentResponse.getDocument();
 		}
 
-		return Optional.empty();
+		return null;
```

Comment:
@LisztLiszt Let's make it a convention: `return null` in the shortcut. Please flip this if-shortcut.